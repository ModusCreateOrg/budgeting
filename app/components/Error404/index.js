// @flow

import React from 'react';
import styles from './style.scss';

const AppError = () => (
  <div className={styles.errorScreen}>
    <div className={styles.errorContainer}>
      <h1 className={styles.errorTitle}>Sorry, item not found.</h1>
      <p> Please check if if you have entered url correctly. </p>
    </div>
  </div>
);

export default AppError;
