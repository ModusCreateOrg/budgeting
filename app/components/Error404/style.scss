.errorScreen {
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100%;
}

.errorContainer {
  display: flex;
  flex-direction: column;
  border: 1px solid lightgrey;
  padding: 15px 30px;
  border-radius: 13px;

  .errorTitle {
    font-size: 24px;
  }
}
