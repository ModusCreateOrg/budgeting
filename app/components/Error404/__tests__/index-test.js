import React from 'react';
import renderer from 'react-test-renderer';
import Error404 from '..';

it('renders correctly', () => {
  const tree = renderer.create(<Error404 />).toJSON();
  expect(tree).toMatchSnapshot();
});
