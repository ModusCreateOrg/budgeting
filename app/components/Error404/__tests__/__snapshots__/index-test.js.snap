// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="errorScreen"
>
  <div
    className="errorContainer"
  >
    <h1
      className="errorTitle"
    >
      Sorry, item not found.
    </h1>
    <p>
       Please check if if you have entered url correctly. 
    </p>
  </div>
</div>
`;
